### ISSUE TITLE
Paint editor keyboard shortcuts

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-paint#38]
Ctrl+c to copy
Ctrl+v to paste (cmd on mac)
Ctrl+a to select all
Esc to deselect all
Ctrl+z to undo
Ctrl+shift+z to redo

**Comment by towerofnix** [comment]
Ctrl+Z / Ctrl+Shift+Z to undo/redo is also a common shortcut. Ctrl+A to select all would be handy. Ctrl+X to cut (delete and copy) would also be useful. (It would be good if shapes could be copied between costumes, but that's a separate issue.)

**Comment by Kenny2github** [comment]
@towerofnix Ctrl-Y to redo is also a thing - consider making both do the same thing?

**Comment by paulkaplan** [comment]
Just a heads up that this is blocked on a bug with scratch-blocks where it listens to keyboard shortcuts while hidden. https://github.com/LLK/scratch-blocks/issues/1019

This is the same issue blocking the sound editor having keyboard shortcuts

**Comment by fsih** [comment]
Ctrl-c

**Comment by fsih** [comment]
Reopening as this was reverted.
Bugs: the keyboard shortcut related bugs at https://github.com/LLK/scratch-gui/issues/3066

**Comment by towerofnix** [comment]
@fsih That google doc is private, should it be public?

**Comment by fsih** [comment]
@towerofnix made it a github link
